txnsync/peer.go

```diff
@@ -378,6 +400,7 @@ func (p *Peer) selectPendingTransactions(pendingTransactions []pooldata.SignedTx
 		if p.recentIncomingBloomFilters[filterIdx].filter == nil {
 			continue
 		}
+		// TODO: should this use all bloom filters for proposal sending?
```

Comment:
should this use all bloom filters for proposal sending?

Reply:
The `p.recentIncomingBloomFilters` would filter out quite many of these.. but the bloom filters would help with the rest. A good starting point would be to try and use all the bloom filters. I don't know if this could be sustained, and/or what the real benefits of that. ( we should be measuring that to figure out if this is something we should use or not )